code/modules/admin/verbs/buildmode.dm

```diff
@@ -293,7 +293,7 @@
 					var/turf/T = get_turf(object)
 					log_admin("Build Mode: [key_name(user)] modified [T] ([T.x],[T.y],[T.z]) to [objholder]")
 					T.ChangeTurf(objholder)
-				else
+				else if(objholder)
```

Comment:
[Unless I am missing something, this should not be null](https://github.com/tgstation/tgstation/blob/ca32b8efba568139b2e3c779caf5c72e93cd2da0/code/modules/admin/verbs/buildmode.dm#L165)